src/astx/callables.py

```diff
@@ -386,3 +386,32 @@ def get_struct(self, simplified: bool = False) -> ReprStruct:
         key = "YIELD-EXPR"
         value = {} if self.value is None else self.value.get_struct(simplified)
         return self._prepare_struct(key, value, simplified)
+
+
+@public
+@typechecked
+class YieldFromExpr(Expr):
+    """AST class for YieldFromExpr."""
+
+    value: Optional[Expr]
```

Comment:
```suggestion
    value: Expr
```

value should be required.